package/cpp/rnskia/dom/base/JsiDomNode.h

```diff
@@ -73,9 +73,29 @@ class JsiDomNode : public JsiHostObject,
   }
 
   /**
-   Empty setProp implementation - compatibility with JS node
+   Updates the selected property value
    */
-  JSI_HOST_FUNCTION(setProp) { return jsi::Value::undefined(); }
+  JSI_HOST_FUNCTION(setProp) {
+    if (_propsContainer == nullptr) {
+      // TODO: we ignore individual properties updates if the initial properties
+      // hasn't been defined. It is likely an error if we reach this branch and
+      // perhaps should throw an exception but platformContext isn't available
+      // here.
+      return jsi::Value::undefined();
+    }
+    auto propName = arguments[0].asString(runtime).utf8(runtime);
+    const jsi::Value &propValue = arguments[1];
+
+    auto mappedProps = _propsContainer->getMappedProperties();
+    auto propMapIt = mappedProps.find(JsiPropId::get(propName));
+    if (propMapIt != mappedProps.end()) {
+      for (auto &prop : propMapIt->second) {
+        prop->updateValue(runtime, propValue);
```

Comment:
This doesn't appear to work if the type is Float32Array, is it because it only exists in one of the JS runtime?